### PR TITLE
more papers, and some corrections

### PR DESCRIPTION
**Pull request by darijgr**

### CODE REVIEW

**Comment by haraldschilly** [comment]
Travis fails, see error in the details above.

**Comment by darijgr** [comment]
I'm afraid I can't debug this; is there a way to see the line number?

The reason for removing the {}s is that your site doesn't seem to correctly parse \mathbb{...}'s; it appears to just throw away the {} signs, leaving unparseable things like "\mathbbZ" behind. But maybe the whitespace solution is just as bad.

**Comment by haraldschilly** [comment]
Hmm, I don't know how this "mathbb" is handled. There is a parser script `pubparse.py` and it does some conversions for the website. I even found `mathbb` somewhere in a replacement table. I might have touched it once or twice, but I'm not sure what it does. What you describe sounds like a bug report.

**Comment by haraldschilly** [comment]
well, I think I'll just try to merge this and fix whatever is going on. Probably putting back the curly brackets. Then in a second iteration we can look into what's going on with the formulas.

File: bibliography-sage.bib [hunk 1]
```diff
@@ -1837,12 +1837,20 @@ @article{FiteEtAl2012
...
   title        = {On the Number of Carries Occuring in an Addition $\mod 2^k-1$},
+  volume       = {12},
+  ISSN         = {1867-0652},
+  url          = {http://dx.doi.org/10.1515/integers-2011-0124},
+  DOI          = {10.1515/integers-2011-0124},
+  number       = {4},
+  journal      = {Integers},
+  publisher    = {Walter de Gruyter GmbH},
+  year         = {2012},
+  month        = {Jan}
```
Comment (haraldschilly) [inline bibliography-sage.bib]: btw, in order to spot the problem, I ran biber to format the bib file (that's the formatter on cocalc for bib files). it reported: 

```
ERROR - BibTeX subsystem: /tmp/qKkIbDXkQB/tmp-106KYl7WqWHM7F.tmp_33083.utf8, line 1852, syntax error: found "date", expected end of entry ("}" or ")") (skipping to next "@")
INFO - WARNINGS: 3
INFO - ERRORS: 1
```

I'll take care

**Comment by haraldschilly** [comment]
ok, merged, and ready for round 2 :-)

**Comment by darijgr** [comment]
Thanks Harald for tracing this one back to the typo! I wasn't aware of
biber; now I'm wondering if I should use it for my own papers (have been
avoiding bibtex so far for its pitfalls and inflexibility).

**Comment by haraldschilly** [comment]
If you're curious, here is how [cocalc calls biber as a tool](https://github.com/sagemathinc/cocalc/blob/master/src/smc-project/formatters/bib-format.ts#L13).

and please be aware that it canonicalized many fields as well.   e.g. your year and month became `date = {2011-01},` I don't know much about the syntax details and their history, but I just blindly trust that it knows what it is doing :walking_man: … not sure if this is still bibtex compatible.

**Pull request merged**